spec/lib/topic_view_spec.rb

```diff
@@ -45,10 +48,10 @@
   it "should preload comments, comments count and user voted status correctly" do
```

Comment:
I think there was some limit and order in your query. Not sure if you feel if it's necessary enough to test them, if you don't feel it's necessary, that's okay too. Sometimes unfortunately it's not obvious what "correctly" is T_T

Reply:
Good point :) Now that you mentioned it, I've tried to be more descriptive in my spec descriptions as well. I've added an additional test case for the limit.